pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = [
+    "setuptools>=45",
+    "setuptools_scm>=7", 
+    "tomli>=1.0.0; python_version < '3.11'",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+write_to = "dsps/_version.py"
+
+[tool.setuptools.package-data]
```

Comment:
You need to include a bunch of setuptools hooks to make the package data work. See the docs here: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html